.github/workflows/peribolos-schema-validate.yaml

```diff
@@ -25,11 +25,11 @@ jobs:
     name: Check peribolos schema.
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v3
+    - uses: actions/checkout@v4
       with:
         fetch-depth: 0
         lfs: true
-    - run: pip install strictyaml
+    - run: pip install strictyaml --break-system-packages
```

Comment:
Let's open an issue to properly investigate this for all deps? Solutions are:
- use this flag as a workaround
- use virtual env
- use system package manager